examples/va/nfv/sriov/edpm/values.yaml

```diff
@@ -1,89 +1,19 @@
-#
-# CHANGEME:
-#
-# - Change the first interface name specified in "spec.nodeTemplate.ansible.ansibleVars.edpm_network_config_template"
-#   to be the control plane interface on your compute nodes
-#     i.e.
-#     members:
-#     - type: interface
-#       name: enp7s0    <---- change
-# - Change the second interface name specified in "spec.nodeTemplate.ansible.ansibleVars.edpm_network_config_template"
-#   to be the SRIOV interface on your compute nodes
-#     i.e.
-#     - type: sriov_pf
-#       name: eno4      <---- change
-# - Change "spec.nodeTemplate.ansible.ansibleVars.edpm_sshd_allowed_ranges[0]" to
-#   match the CIDR of your control plane network, if necessary
-# - Change "edpm_bootstrap_command" to include subscription-manager and podman login
-#   commands to register the EDPM nodes as in https://access.redhat.com/solutions/253273
-# - Change the "spec.nodes[].ansible.ansibleHost" and "spec.nodes[].networks[0].fixedIP" values
-#   to match the control plane IPs of your nodes, if necessary
-# - Change "spec.nodeTemplate.ansible.ansibleVars.edpm_kernel_args" to be the desired kernel
-#   args for your environment
-# - Change "spec.nodeTemplate.ansible.ansibleVars.edpm_tuned_isolated_cores" to be the desired
-#   CPU core isolation for your environment
-# - Change "spec.nodeTemplate.ansible.ansibleVars.edpm_neutron_sriov_agent_SRIOV_NIC_physical_device_mappings"
-#   to the desired mapping of a network name to your SRIOV interface
-#
-
-apiVersion: dataplane.openstack.org/v1beta1
-kind: OpenStackDataPlaneNodeSet
+# local-config: referenced, but not emitted by kustomize
+apiVersion: v1
+kind: ConfigMap
```

Comment:
i woudl genereal consier manualy defineing a configmap instead of using the configmap generator to be anti pattern by the way.

it certenly works but i don't thinkthis is the way kustomize is in intended to be used.
with that said i have not used kustomize extensively by my initial feel ing is this feels like we are sidetpping the inbuilt tools.